utils/businesslogic.py

```diff
@@ -50,24 +47,93 @@ def updateuser(user):
         for invoice in invoices:
             try:
                 transaction = BankTransaction.objects.get(reference_number=invoice.reference_number, has_been_used=False)
-                BusinessLogic.check_transaction_pays_custominvoice(transaction)
+                BusinessLogic._check_transaction_pays_custominvoice(transaction)
             except BankTransaction.DoesNotExist:
                 pass
 
-        # Now we assume the transaction is done to user's default account
+        # Now we check transactions done to user's default account
         defaultservice = MemberService.objects.get(id=settings.DEFAULT_ACCOUNT_SERVICE)
-
-        # Update user's servicesubscriptions
         servicesubscriptions = ServiceSubscription.objects.filter(user=user, service=defaultservice)
 
+        # Update user's servicesubscription if it exists:
         if len(servicesubscriptions) == 1:
             for subscription in servicesubscriptions:
                 print('Examining default subscription', subscription)
-                BusinessLogic.updatesubscription(user, subscription, servicesubscriptions)
-                BusinessLogic.check_servicesubscription_state(subscription)
+                BusinessLogic._updatesubscription(user, subscription, servicesubscriptions)
+                BusinessLogic._check_servicesubscription_state(subscription)
+
+    """
+    Rejects a membership application and deletes the user
+    """
+    @staticmethod
+    def reject_application(application):
+        print('Rejecting app ', application)
+        # TODO: Send mail and any other notifications to user?
+        # Should delete the application
+        application_denied.send(sender=application.__class__, instance=application)
+        application.user.delete()
+
+    """
+    Accepts a membership application
+    """
+    @staticmethod
+    def accept_application(application):
+        print('Accepting app ', application)
```

Comment:
if you move the comment under the method then the docstring will be shown correctly in editors.

Like this:
![image](https://user-images.githubusercontent.com/1352132/75665108-4fa2f080-5c7c-11ea-961f-77cbd3f701a3.png)


Then editors can show it:
![image](https://user-images.githubusercontent.com/1352132/75665081-43b72e80-5c7c-11ea-85f9-2808c4346d62.png)


Currently it just shows this:
![image](https://user-images.githubusercontent.com/1352132/75665162-69443800-5c7c-11ea-8c28-339dbaad955f.png)

https://www.python.org/dev/peps/pep-0257/